README.md

```diff
@@ -40,6 +40,8 @@ Run Label Studio in a Docker container and access it at `http://localhost:8080`.
 
 ```bash
 docker pull heartexlabs/label-studio:latest
+mkdir -p mydata/label-studio/data
```

Comment:
by the way maybe it's best to just create mydata without additional path? wouldn't it be after mounting `/label-studio/data/label-studio/data` in the container?

Reply:
@nikelborm I thought so too, but I just copied what was in the readme before... I can change that easily if that is what blocks this from being merged.

Reply:
I agree, it feels strange.